src/main/kotlin/no/nav/syfo/model/PdfModel.kt

```diff
@@ -8,3 +13,26 @@ data class PdfModel(
     val navnSignerendeLege: String,
     val antallVedlegg: Int,
 )
+
+const val BYTE_ORDER_MARK = '\uFEFF'
+const val INVALID_CHAR = '\uFFFE'
+const val NUL = 0.toChar()
+
+val illegalCharacters = listOf(BYTE_ORDER_MARK, INVALID_CHAR, NUL)
+
+fun PdfModel.sanitizeForPdfGen(): PdfModel {
+    val pdfModelJsonString = objectMapper.writeValueAsString(this)
+
+    val sanitizedJson = pdfModelJsonString.toCharArray().filter {
+        when (it) {
+            in illegalCharacters -> {
```

Comment:
Hmm, nå får vi vel ikke sjekket om noe er mindre enn 0? Må det ikke bli noe ala `in illegalCharacters || it >= NIL`? 🤔